TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/commands/Com_SetShootPower.java

```diff
@@ -0,0 +1,20 @@
+package org.firstinspires.ftc.teamcode.subsystems.commands;
+
+import com.arcrobotics.ftclib.command.CommandBase;
+
+import org.firstinspires.ftc.teamcode.subsystems.ShooterSystem;
+
+public class Com_SetShootPower extends CommandBase {
+
+    private final ShooterSystem shooterSystem;
+
+    public Com_SetShootPower(ShooterSystem subby){
+        shooterSystem = subby;
+        addRequirements(shooterSystem);
+    }
+    @Override
+    public void execute(){
+        shooterSystem.adjust();
+    }
+
+}
```

Comment:
This class is useless. You don't even use it. Plus you should just follow my previous advice.